arrow-core/src/main/kotlin/arrow/core/extensions/sequence/alternative/SequenceKAlternative.kt

```diff
@@ -148,8 +138,7 @@ fun <A> Sequence<A>.optional(): Sequence<Option<A>> =
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "guard(arg0)",
-  "arrow.core.extensions.sequence.alternative.Sequence.guard"
+    "if (arg0) sequenceOf(Unit) else emptySequence()"
```

Comment:
should we be promoting in these replacements apis we are providing such as `Sequence.unit` otherwise we should question if `Sequence.unit` has value over `sequenceOf(Unit)`?

Reply:
We can't do `Sequence.unit` since we don't own `Sequence`.

I've used `Either.unit()` a lot, at least I use it frequently if I've needed to use `Either` in some kind of loop. In general, however, I'd say that it's probably not very useful.